app/client-side-scripts/multi-select.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+// TODO: we should probably do some browser testing in this project to prove this all works as intended
+
+// NPM Dependencies
+const $ = require('jquery')
+
+// Local Dependencies
+const multiSelect = require('../views/includes/multi-select.html')
+
+// Variables
+const MAXIMUM_VISIBLE_ITEMS = 5.5 // Maximum amount of items to show in dropdown
+// Selectors
+const DOCUMENT_SELECTOR = window.document
+const ENHANCEMENT_SELECTOR = 'select[data-enhance-multiple]'
+const TOP_LEVEL_SELECTOR = 'div.multi-select'
```

Comment:
Don't use a qualifying tag name for selectors, it’s slows jQuery down. Make the class more specific is necessary

Reply:
fixed